 polls/urls.py  | 1 +
 polls/views.py | 4 ++++
 2 files changed, 5 insertions(+)

diff --git a/polls/urls.py b/polls/urls.py
index 1e17229..316d1d3 100644
--- a/polls/urls.py
+++ b/polls/urls.py
@@ -7,4 +7,5 @@ urlpatterns = patterns('',
     url(r'^(?P<pk>\d+)/$', views.DetailView.as_view(), name='detail'),
     url(r'^(?P<pk>\d+)/results/$', views.ResultsView.as_view(), name='results'),
     url(r'^(?P<question_id>\d+)/vote/$', views.vote, name='vote'),
+    url(r'^create/$', views.QuestionCreateView.as_view(), name='create'),
 )
diff --git a/polls/views.py b/polls/views.py
index 83a2054..e58280c 100644
--- a/polls/views.py
+++ b/polls/views.py
@@ -44,6 +44,10 @@ class DetailView(generic.DetailView):
 class ResultsView(LoginRequiredMixin, generic.DetailView):
     model = Question
     template_name = 'polls/results.html'
+
+class QuestionCreateView(generic.CreateView):
+    model = Question
+    success_url = '/polls'
 	
 def vote(request, question_id):
     p = get_object_or_404(Question, pk=question_id)
